package.json

```diff
@@ -34,7 +34,7 @@
     "logops": "2.1.0",
     "underscore": "~1.7.0",
     "revalidator": "~0.3.1",
-    "mongoose": "5.7.7",
+    "mongoose": "5.11.18",
```

Comment:
Should be noted in CHANGES_NEXT_RELEASE, this way:

```
- Upgrade mongoose dependency from 5.7.7 to 5.11.18 (#204)
```

Reply:
Fixed 31dfddf